nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/ClientRequest.java

```diff
@@ -98,6 +99,32 @@ default B header(Http.HeaderName name, String value) {
      */
     B headers(Function<ClientRequestHeaders, WritableHeaders<?>> headersConsumer);
 
+    /**
+     * Accepted media types. Supports quality factor and wildcards.
+     *
+     * @param accepted media types to accept
+     * @return updated request
+     */
+    default B accept(HttpMediaType... accepted) {
+        return headers(it -> {
+            it.accept(accepted);
+            return it;
+        });
+    }
+
+    /**
+     * Sets the MIME type of the response body.
+     *
+     * @param contentType Media type of the content.
+     * @return updated request
+     */
+    default B contentType(HttpMediaType contentType) {
```

Comment:
The javadoc is wrong - this sets the content type of the request. Accept header is the one to set expected response types.

Reply:
Updated javadoc